images/CubesterYT/KeySimulation.svg

```diff
@@ -0,0 +1,698 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<!DOCTYPE svg PUBLIC "-//W3C//DTD SVG 1.1//EN" "http://www.w3.org/Graphics/SVG/1.1/DTD/svg11.dtd">
+<svg version="1.1" id="Layer_1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px" width="600px" height="300px" viewBox="0 0 600 300" enable-background="new 0 0 600 300" xml:space="preserve">  <image id="image0" width="600" height="300" x="0" y="0"
+    href="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAlgAAAEsCAIAAACQX1rBAAABAGlDQ1BpY2MAABiVY2BgPMEABCwG
```

Comment:
Why made the thumbnail an SVG if it's just a PNG internally?

Reply:
If the thumbnail is a PNG don't make the effort to convert it to an SVG